application/src/main/kotlin/no/kartverket/matrikkel/bygning/application/models/Egenregistrering.kt

```diff
@@ -95,3 +100,168 @@ data class Egenregistrering(
     val prosess: ProsessKode,
     val bruksenhetRegistrering: BruksenhetRegistrering,
 )
+
+sealed interface EgenregistreringBase {
+    val id: EgenregistreringId
+    val eier: Foedselsnummer
+    val registreringstidspunkt: Instant
+    val prosess: ProsessKode
+    val bruksenhetId: BruksenhetBubbleId
+    val type: FeltRegistreringType
+}
```

Comment:
```suggestion
sealed interface EgenregistreringBase {
    val id: EgenregistreringId
    val eier: Foedselsnummer
    val registreringstidspunkt: Instant
        get() = Instant.now()
    val prosess: ProsessKode
    val bruksenhetId: BruksenhetBubbleId
    val type: FeltRegistreringType
}
```

kan vel sette registreringstidspunkt sånn så slipper du å override og sette på alle klassene som arver